src/commands/synthetics/index.ts

```diff
@@ -0,0 +1,5 @@
+export {CiError} from './errors'
+export * from './interfaces'
```

Comment:
Should we also namespace as `types` maybe?
```suggestion
export * as types from './interfaces'
```

Reply:
What would that change? Wouldn't that just be more tedious to reference?

Reply:
Discoverability and general organisation of the library's API.

In the end, I'm not even sure we need these `index.js` files and we should simply import from the actual arborescence, which is less confusing (there is not a secondary, somewhat duplicated API to access some part of the code), less maintenance (you don't have to maintain a public API, it's relying on the current arbo), less documentation (you don't have to document what's exported to the public API and what's not, you only document what's actually usable as a library)...

For instance, I think this is clearer:
```javascript
const {executeTests} = require('@datadog/datadog-ci/commands/synthetics/run-test');
const Reporter = require('@datadog/datadog-ci/commands/synthetics/interfaces');

await executeTests();
const report: synthetics.Reporter = new synthetics.DefaultReporter();
```

The less we try to explicitly control what's exported, the easier it will be for us to maintain and the more freedom the user will have.

Reply:
This is being discussed with @m-rousse and @sdeprez on Slack.
I'll follow along their take on this. 
Just wanted to be sure that we present something clean and explicit to the customer.

Reply:
FTR as discussed on slack, we prefer exposing only functions that should be used by users of the package.